things/groups/api/logging.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"time"
 
+	mfgroups "github.com/mainflux/mainflux/internal/mainflux/groups"
 	mflog "github.com/mainflux/mainflux/logger"
 	"github.com/mainflux/mainflux/things/groups"
```

Comment:
We can unify the `things/groups` service and `users/groups` service

Reply:
Page struct is a bit different and that would still result in two interfaces